js/ui/tooltips.js

```diff
@@ -1,20 +1,17 @@
 const Lang = imports.lang;
 const Mainloop = imports.mainloop;
 const St = imports.gi.St;
+const Gio = imports.gi.Gio;
 
 const Applet = imports.ui.applet;
 const Main = imports.ui.main;
 const SignalManager = imports.misc.signalManager;
 const Tweener = imports.ui.tweener;
-const Gio = imports.gi.Gio;
+const {PanelLoc} = imports.ui.panel;
```

Comment:
CJS will complain about exported variables with lexical scoping since [PanelLoc in panel.js](https://github.com/germanfr/Cinnamon/blob/ef1e87fe642b6d9991484bc33d78d422baf4ee45/js/ui/panel.js#L69) is declared with `const`. Changing it to `var` would fix it.

Reply:
@jaszhix done :+1: